.travis.yml

```diff
@@ -63,6 +63,9 @@ script:
   - if [[ "$WERROR" != "true" ]]; then
       python build_total.py ecl || travis_terminate 1;
     else
+      git clone https://github.com/Statoil/libecl.git || travis_terminate 1;
```

Comment:
Hmmm - what was the purpose of this - the `libecl` repository has already been cloned and postioned at the right commit by Travis?